src/systems/pad.cpp

```diff
@@ -19,6 +21,75 @@ static mkb::PadStatusGroup s_pad_status_groups[4];
 
 static u8 s_dir_down_time[8];
 
+static mkb::PADStatus s_original_inputs[4];
+
+void get_merged_stick(StickInputs& out) {
+    out = {};
+    if (s_exclusive_mode) return;
+
+    for (u32 i = 0; i < LEN(mkb::pad_status_groups); i++) {
+        mkb::PADStatus& status = mkb::pad_status_groups[i].raw;
+        if (status.err == mkb::PAD_ERR_NONE) {
+            out.x += status.stickX;
+            out.y += status.stickY;
+        }
+    }
+
+    out.x = CLAMP(out.x, -128, 127);
+    out.y = CLAMP(out.y, -128, 127);
```

Comment:
Can we accumulate these inputs in `pad::tick()` instead of in the getters? It'll deduplicate some code here

Reply:
Yup, cleaned it up a bit, let me know if there's anything else I should change!